src/mcover/coverage/client/LcovPrintClient.hx

```diff
@@ -41,26 +41,35 @@ class LcovPrintClient implements CoverageReportClient {
 
 	var testName:String;
 	var lcovFileName:String;
+	var lastKnownAbsoluteBasePath:String;
 
 	public function new(name:String, ?fileName:String) {
 		testName = name;
-		if (fileName == null) 
+		if (fileName == null)
 		{
 			lcovFileName = "lcov.info";
-		} 
+		}
 		else
 		{
 			lcovFileName = fileName;
 		}
 	}
 
 	public function report(coverage:Coverage) {
-		sys.io.File.saveContent(lcovFileName, makeLine("TN", testName) + "\n");
+		sys.io.File.saveContent(lcovFileName, "\n");
 
+		var fileClassMap:Map<String, Array<Clazz>>=new Map<String, Array<Clazz>>();
 		for (cls in coverage.getClasses()) {
-			reportClass(cls);
+			var path = getAbsolutePath(cls);
+			if (fileClassMap.exists(path)) {
+				fileClassMap.get(path).push(cls);continue;
```

Comment:
`continue` on next line

Reply:
I guess I'm used to having a formatter clean up my messy code... :)